index.js

```diff
@@ -19,3 +16,6 @@ lang.translations = Object.assign(
     [key]: resources[key].translation,
   }))
 );
+
+// eslint-disable-next-line import/no-commonjs
+require('./src/App');
```

Comment:
When `import` was used (even when placed here at the bottom of the file), I noticed that it would actually get bundled before the `lang` setup was executed. This created some issues in places where translations were evaluated outside of components, such as in setting page titles. Changing this to `require` fixed that issue, but let me know if there are any downsides to this change or if there is a better way to handle this.